types/pouchdb-mapreduce/index.d.ts

```diff
@@ -10,15 +10,19 @@
 /* tslint:disable:no-single-declare-module */
 
 declare namespace PouchDB {
+    type Map<Content extends {}> = (doc: Content) => void;
```

Comment:
We also have the "safe" version of this provided by PouchDB:
```ts
type SafeAndSlowerMap<Content extends {}> = (doc: Content, emit: (key, value) => {}) => void;
```
Might be of value to add it: https://pouchdb.com/api.html#closures